docs/release-notes.md

```diff
@@ -1,230 +1,124 @@
-- **api:** hardware information ([#460](https://github.com/open-amt-cloud-toolkit/mps/issues/460)) (#48f4576) 
-- **cira:** fix cira channel distruption across multiple API Calls (#ef4fdb8) 
-- **cira:** remove javascript libraries and leverage new refactored AMT libraries (#fa68566) 
-- **logging:** centralizes logging and mqtt messages into messages.ts (#de32b04) 
-- **logging:** fixes spelling mistakes (#d0f9a4b) 
-- **mps:** remove node-vault (#9d4963e) 
-- **power:** use new device handling (#27c5c28) 
-- **websockets:** KVM and SOL now use new CIRA connection (#e075539) 
-- **wsman:** use new dependency wsman-message (#2f0c65c) 
-- see change log for full list of changes
-#### RPC-Go
-- **activate:** prompts for password when device is activated
-- **cli:** Remove ControlModeRaw from json
-- **cli:** Add json flag to version command
-- **log:** adds -json option to all rpc commands
-- **output:** adds json output for amtinfo command
-- update password with user input
-- **pthi:** converts amt and pthi commands to go from C
-- see change log for full list of changes
-#### Sample Web UI
-- add-dialog: uses rpc-go syntax (#c729709)
-- cypress: consolidate http codes (#0ebdb3f)
-- see change log for full list of changes
-#### UI Toolkit
-- upgrade ws from 8.4.2 to 8.5.0 (#228166b)
-- **npm:** add job to publish to npm (#33bdf7e) 
-- **semantic-release:** adds automated releases (#fde186b)
-- see change log for full list of changes
-#### WSMAN-MESSAGES
-*Expect this set of libraries to version up very fast as we continue to enhance its capabilities to support the rest of the Open AMT Cloud Toolkit.  Since this is new, adding the full set of changes since 1.0.0.*
-
-v2.1.0 - 2022-03-15: 
-
-- **amt:** add wifiportconfiguration service PUT and GET calls (#b9767cf)
-
-v2.0.1 - 2022-03-02: 
-
-- **AMT:** update wifi_port naming convention ([#86](https://github.com/open-amt-cloud-toolkit/wsman-messages/issues/86)) (#4d2830d) 
+<p class="divider"></p>
+## Note From the Team
 
-v2.0.0 - 2022-03-02: 
+Hey everyone,
 
-- **tls-configuration:** add support for TLS Connection type (#0bcddb6) 
-- **BREAKING CHANGE:** messageID is no longer passed in to functions and is tracked internally
+It has been a very quick 6 weeks and the team is very excited to deliver Open AMT Cloud Toolkit version 2.3.  Unlike previous releases, every component of the Toolkit has been updated in some way.  If you haven't had a chance yet, I encourage you to watch the release video where Mike provides some highlights from this release.  Additionally, we've added a [Video Walkthroughs](https://open-amt-cloud-toolkit.github.io/docs/2.3/videos/) section to our documentation where Bryan Wendlandt has been creating video tutorials of our [Getting Started Guide](https://open-amt-cloud-toolkit.github.io/docs/2.3/GetStarted/prerequisites/).  Watch that space for additional video content to be delivered there.  Don't forget to like and subscribe!
```

Comment:
just confirming, you're asking them to like and subscribe to Intel? They won't be notified of any toolkit videos since they are unlisted -- in fact they'll likely be spammed of notifications nothing to do with the toolkit.

Reply:
ok, fine.  I removed it.  so much for me trolling....